test_runner/regress/test_disk_usage_eviction.py

```diff
@@ -270,7 +292,14 @@ def test_broken_tenants_are_skipped(eviction_env: EvictionEnv):
     env.neon_env.pageserver.allowed_errors.append(".*" + GLOBAL_LRU_LOG_LINE)
 
 
-def test_pageserver_evicts_until_pressure_is_relieved(eviction_env: EvictionEnv):
+@pytest.mark.parametrize(
+    "order",
+    [EvictionOrder.ABSOLUTE_ORDER, EvictionOrder.RELATIVE_ORDER_EQUAL],
+    ids=["absolute", "relative"],
+)
```

Comment:
@bayandin specifying the `ids` is probably extra since the EvictionOrder inherits from str?

Reply:
Yep, it should be fine without it

Reply:
e6bf6e08b